lightkurve/targetpixelfile.py

```diff
@@ -1050,3 +1051,113 @@ def _make_aperture_extension(self):
                                       template.comments[kw])
         hdu.header['EXTNAME'] = 'APERTURE'
         return hdu
+
+class TargetPixelFileCollection(object):
+    """
+    Collects multiple TPF objects together with helpful functions.
+    """
+    def __init__(self, targetPixelFiles):
+        try:
+            tpfs = np.asarray(targetPixelFiles)
+        except:
+            raise TypeError("Unable to parse input")
```

Comment:
I don't understand where this error will ever occur, if we always check they are TPFs then the collection should always work?

Reply:
I imagined a scenario where the input wasn't something that could be split using the np.asarray method.